lib/Installation/ModuleSelection/ModuleSelectionPage.pm

```diff
@@ -41,13 +41,25 @@ sub is_shown {
 
 sub get_modules {
     my ($self) = @_;
-    my @modules = ($self->{rt_items}->text() =~ /<a href='(.*?)'.*<\/a>/g);
+    my @modules = ($self->{rt_items}->text() =~ /<a href='(.*?)-\d+/g);
+    return \@modules;
+}
+
+sub get_modules_textmode {
+    my ($self) = @_;
+    my @modules = ($self->{rt_items}->text() =~ /<a href="(.*?)-\d+/g);
     return \@modules;
 }
 
 sub get_selected_modules {
     my ($self) = @_;
-    my @modules = ($self->{rt_items}->text() =~ /<a href='(.*?)'.*inst_checkbox-on.*<\/a>/g);
+    my @modules = ($self->{rt_items}->text() =~ /<a href='(.*?)-\d+.*inst_checkbox-on.*<\/a>/g);
+    return \@modules;
+}
+
+sub get_selected_modules_textmode {
```

Comment:
I really like it that at the end we can only have one test, make it easy to search what we test, not distinguishing by ncurses or qt. Could you create a ticket to improve this part in the future with those advices offline about `Mojo::DOM`, I already noticed that in the past, that we need to use some html parser instead of trusting in regex.